tests/test_models.py

```
@@ -0,0 +1,78 @@
+from pathlib import Path
+
+try:
+    from pydantic import ValidationError
+except ModuleNotFoundError:
+    ValidationError = None
+
+if ValidationError is not None:
+    MODELS_PATH = Path(__file__).resolve().parents[1] / "app" / "models.py"
+    spec = importlib.util.spec_from_file_location("models", MODELS_PATH)
+    models = importlib.util.module_from_spec(spec)
+    assert spec and spec.loader
```

Ruff (0.15.20)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)